components/editor/utils/JSONField.js

```diff
@@ -65,7 +67,7 @@ class JSONField extends Component {
       <Field
         label={label}
         value={
-          stateValue === undefined ? JSON.stringify(value, null, 2) : stateValue
+          stateValue === undefined ? JSON.stringify(value, null, 4) : stateValue
```

Comment:
(Although we work with 2 spaces)

Reply:
fand das sah irgendwie zu gedrängt aus...

Reply:
Hilft aber beim Copy-Paste in und aus dem Codeeditor.

Reply:
jup habs schon reverted.